aws/resource_aws_emr_security_configuration_test.go

```diff
@@ -58,21 +58,20 @@ func testAccCheckEmrSecurityConfigurationDestroy(s *terraform.State) error {
 		resp, err := conn.DescribeSecurityConfiguration(&emr.DescribeSecurityConfigurationInput{
 			Name: aws.String(rs.Primary.ID),
 		})
-		if err == nil {
```

Comment:
Invert logic here to match preferred style across codebase of error checking first.